### DISCUSSION TITLE
Systrem Requirements

### DISCUSSION THREAD

**Discussion by ghost** [PowerDNS/pdns#13862] [Q&A]
I've looked and can not find system requirement

**Comment by rgacogne** [comment]
I'm afraid you will have to type more words to explain what you are looking for, otherwise we cannot really help you. At the bare minimum you might want to specify which product you are interested in (Authoritative Server, Recursor or DNSdist) and what the use-case is.

**Comment by ghost** [comment]
PowerDNS Authoritative Server

I am trying to run my own name servers and continue to get the same error that when I try to install powertools, 
Rocky 9 Linux Error: No matching repo to modify: PowerTools.
I've have this in repo list:
repo id                                                            repo name
appstream                                                          Rocky Linux 9 - AppStream
baseos                                                             Rocky Linux 9 - BaseOS
crb                                                                Rocky Linux 9 - CRB
epel                                                               Extra Packages for Enterprise Linux 9 - x86_64
epel-cisco-openh264                                                Extra Packages for Enterprise Linux 9 openh264 (From Cisco) - x86_64
extras                                                             Rocky Linux 9 - Extras
nodesource-nodejs                                                  Node.js Packages for Linux RPM based distros - x86_64
yarn                                                               Yarn Repository

What am I missing? I can't find system requirements or which OS is recommended
I'm trying to set up 3 name server
I'm a 1 person team and I'm open to help

**Comment by ghost** [comment]
I got PDNS installed but after I logged out I can not log back in and using the forgot password doesn't send email

**Reply by Habbie** [reply]
pdns does not have a login screen of any kind. What product are you talking about?

**Comment by omoerbeek** [comment]
My guess is that @440music is using one of the web based admin front ends to the authoritative server. It should be noted that these are made by third parties, not by PowerDNS.

**Comment by ghost** [comment]
I have PowerDNS working although I've encountered another challenge that I've not found the fix yet.
The login page loads and I see: Powered by [PowerDNS-Admin](https://powerdnsadmin.org/) I can not login and I know the user/password are correct I get this message:
Please confirm your email address first 
I don't get the authentication or Reset Password email, is it possible to authenticate or reset the admin password without the email?

**Reply by omoerbeek** [reply]
Your question has been answered multiple times: PowerDNS-Admin is not a PowerDNS product, try to get help from the people creating it.
